### PR TITLE
Portfolio-Desktop version

### PR DESCRIPTION
**Pull request by maryam0007**
**In this PR**

- I have applied media queries for desktop version of portfolio
- I have added images for the desktop version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,29 +17,37 @@
...
+        <div class="accounts-flex">
         <ul class="accounts">
           <li>
             <a href="#"><img src="/images/github.png" alt=""></a>
           </li>
           <li>
-            <a href="#"><img src="/images/linkedin.png" alt=""></a>
+          <a href="#"><img src="/images/linkedin.png" alt=""></a>
+
           </li>
```
Comment (algerina) [inline index.html:51]: - [ ] You did a nice job however,   the social media icons are not displayed on the desktop version as per the Figma design.

Your design
port3

Figma design
figma

File: index.html [hunk 2]
```diff
@@ -166,50 +178,170 @@ <h1 class="card-heading">Professional Art Printing Data</h1>
...
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul class="card-languages-list">
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
```
Comment (algerina) [inline index.html:199]: - [ ] You should stick to the Figma design as much as possible, the spacing in the card id different than the one in the Figma design, kindly change especially between the "card-languages-list" and the "card-text".

Your design
port2

Figma design
figma2

File: index.html [hunk 3]
```diff
@@ -17,29 +17,37 @@
...
           <ul class="navbar-flex">
-            <li class="nav-logo"><a href="#">Maryam</a></li>
-            <li class="nav-logo">
+            <li class="nav-link"><a href="#">Portfolio</a></li>
+            <li class="nav-link"><a href="#">About</a></li>
+            <li class="nav-link"><a href="#">Contact</a></li>
+            <li class="envelope"><a href="#"></a></li>
+            <li class="mob-bars">
               <a href="#"><i class="fa-solid fa-bars"></i></a>
             </li>
```
Comment (algerina) [inline index.html:28]: - [ ] The hamburger button is not displayed in the mobile version, kindly make sure to fix that issue.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @maryam0007 ,

Great job so far! 
However, you still have minor issues to fix.

May I highlight the following: 
- [x] Your project looks awesome👏  ✔️ 
- [x] No Linter errors✔️ 
- [x] Well done implementing most of the requirements✔️ 



## STATUS: CHANGES REQUIRED ♻️ 
PLEASE: check the comments under the review



 
 ## Optional suggestions

 - Please check the comment under the review


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: index.html (hunk 1 above)
Comment (maryam0007) [reply index.html:51]: @algerina They are not showing because I have used images for that. Its visible in my live version. What should I do now?

Comment (algerina) [reply index.html:51]: @maryam0007 You can fix the issue by adding a`.` at the beginning of every image link. I should be something like ``.

**Pull request merged**